app/retail/emails.py

```diff
@@ -552,14 +553,6 @@ def render_new_bounty(to_email, bounties, old_bounties, offset=3, quest_of_the_d
         pass        
 
     upcoming_events = []
-    if upcoming_grant:
-        upcoming_events.append({
-            'event': upcoming_grant,
-            'title': upcoming_grant.title,
-            'image_url': upcoming_grant.logo.url if upcoming_grant.logo else f'{settings.STATIC_URL}v2/images/emails/grants-neg.png',
```

Comment:
didnt make any sense to show grants by their clr calc date IMHO.  

if we want to include trending grants in the future.. we should do it in its own section. @sebastiantf lmk if you waant to work on that.

Reply:
@sebastiantf another thing i could use help on is that the email takes about 2s to calculate. happy to write a v2 ticket where you do the above and also profile/improve the performance of it so we can send 30k/day (scaling up to 100k in the next year)

Reply:
> didnt make any sense to show grants by their clr calc date IMHO.
> 
> if we want to include trending grants in the future.. we should do it in its own section. @sebastiantf lmk if you waant to work on that.

Yeah sure

Reply:
logging the perf issues here https://github.com/gitcoinco/web/issues/6729

a few more things i want to do... 
- add to subject line: upcoming dates, num notifications, num announcements, etc
- what else?? will know more as this ships.

Reply:
grants